doc/source/regression/dwt-idwt.md

```diff
@@ -172,23 +149,41 @@ Remember that only one argument at a time can be `None`:
 
 ```{code-cell}
 ---
-tags: [raises-exception]
+tags: [raises-exception, remove-output]
 ---
 print(pywt.idwt(None, None, 'db2', 'symmetric'))
 ```
 
++++ {"tags": ["jupyterlite_sphinx_strip"]}
+
+```{code-block} python
+:class: pywt-handcoded-cell-output
+Traceback (most recent call last):
```

Comment:
It would be helpful to me if a review could double check the handcoded tracebacks throughout these pages.

Reply:
I checked them and all of them were correct :D